app/src/lib/constants.ts

```diff
@@ -1 +1,3 @@
 export const PAGE_SIZE = 50;
+
+export const GLOBAL_PERMISSIONS = ["packet.push", "packet.read", "packet.run", "user.manage"] as const;
```

Comment:
I don't really think we should make these constants - if we add new permissions, we'd want the front end to pick them up automatically. So we should make an endpoint for those. One for another ticket though, 

For future functionality - when we support adding scoped permissions to users and roles - we should also save and return with the permission definition whether they are global only (like user.manage) or scope-able (like packet.push).

Reply:
yup 100%.. have created ticket for that one